opba-banking-rest-impl/src/test/java/de/adorsys/opba/tppbankingapi/mapper/FacadeResponseBodyToRestBodyMapperTest.java

```diff
@@ -1,17 +1,41 @@
 package de.adorsys.opba.tppbankingapi.mapper;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.google.common.io.Resources;
 import de.adorsys.opba.protocol.api.dto.result.body.AccountListBody;
 import de.adorsys.opba.protocol.api.dto.result.body.AccountListDetailBody;
+import de.adorsys.opba.protocol.api.dto.result.body.TransactionsResponseBody;
 import de.adorsys.opba.tppbankingapi.ais.model.generated.AccountList;
+import de.adorsys.opba.tppbankingapi.ais.model.generated.TransactionsResponse;
 import de.adorsys.opba.tppbankingapi.mapper.generated.TppBankingApiAisController$AccountListFacadeResponseBodyToRestBodyMapperImpl;
+import de.adorsys.opba.tppbankingapi.mapper.generated.TppBankingApiAisController$TransactionsFacadeResponseBodyToRestBodyMapperImpl;
+import lombok.SneakyThrows;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 
+@SpringBootTest(classes = {TppBankingApiAisController$AccountListFacadeResponseBodyToRestBodyMapperImpl.class,
```

Comment:
Maybe scanning some package is more concise, instead of these long class names (less isolation level for this test is actually better)

Reply:
ok